WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormActionsFactory.swift

```diff
@@ -303,9 +303,10 @@ private extension ProductFormActionsFactory {
     func allSettingsSectionActionsForSubscriptionProduct() -> [ProductFormEditAction] {
         let shouldShowReviewsRow = product.reviewsAllowed
         let shouldShowQuantityRulesRow = isMinMaxQuantitiesEnabled && product.hasQuantityRules
+        let editableSubscription = ServiceLocator.featureFlagService.isFeatureFlagEnabled(.subscriptionProducts)
 
         let actions: [ProductFormEditAction?] = [
-            .subscription(actionable: true),
+            editableSubscription ? .priceSettings(editable: true, hideSeparator: false) : .subscription(actionable: true),
```

Comment:
❓ Should we avoid specifying `editable:` parameter as `true` and instead pass the `editable` value of `ProductFormActionsFactory`? 

It looks like that `ProductFormActionsFactory.editable` will be `false` for `.readonly` formType. 
```suggestion
            editableSubscription ? .priceSettings(editable: editable, hideSeparator: false) : .subscription(actionable: true),
```

Reply:
Nice catch 👍 Fixed in 43ad1116bd06a7e46312dcd9b4ec790e29dbb35f.